Core/Src/util/main_task.c

```diff
@@ -76,12 +123,13 @@ void main_task(void* argument) {
                 // If the event flag contains the error flag ECU led is set to yellow
                 isErrorPresent = event_flags & most_significant_error_flag;
                 if (isErrorPresent) {
-                    set_rgb_led(AMARELO, NO_CHANGE);
+                    led_color_response(most_significant_error_flag);
                     osDelay(20);
                 } else {
                     // Clear the thread flag and set ECU led to normal
                     osThreadFlagsClear(most_significant_error_flag);
-                    set_rgb_led(get_global_var_value(SELECTED_MODE).cor, NO_CHANGE);
+                    set_rgb_led(get_global_var_value(SELECTED_MODE).rgbColor, BLINK200,
+                                ONE_COLOR_PATTERN_SIZE);
```

Comment:
```suggestion
                    set_rgb_led(get_global_var_value(SELECTED_MODE).rgbColor, FIXED,
                                ONE_COLOR_PATTERN_SIZE);
```
O led não pisca com o carro em RTD e sem erro